config/env.development

```diff
@@ -93,8 +93,9 @@ IMAGE_URL=http://localhost/v1/image
 USERS_PORT=6666
 
 # Users Service URL
-USERS_URL=http://localhost/v1/user
-FIRESTORE_EMULATOR_HOST=localhost:8088
+USERS_URL=http://localhost/v1/users
+FIRESTORE_EMULATOR_HOST=firebase:8088
```

Comment:
Should this not be `localhost:8088`, since we expose it on the host outside of docker?

Reply:
`localhost` does not point to the same domain between two containers, with this set to localhost the users docker container cannot talk to the firebase docker container when running in dev mode using Docker.